api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -161,4 +162,25 @@ private void validateStopDatetime(Visit visit, Visit otherVisit, Errors errors)
 		}
 		
 	}
+
+	/**
+	 * Validate Birthdate in case where it is estimated
+	 */
+	public void validateBirthDate(Visit visit, Errors errors) {
```

Comment:
this method should not be public it should be private and work like for example `validateStopDatetime()` and be used in the public method `validate()` which is the `Validator`'s public API.

your tests than test the `validate()` method as do the existing tests.